manifests/python.yml

```diff
@@ -55,6 +55,12 @@ tests/:
           fastapi: v2.5.0.dev1
           flask-poc: v2.1.0.dev
           python3.12: v2.1.0.dev
+        Test_Schema_Response_Body_env_var:
+          '*': missing_feature
+          django-poc: v2.6.0.dev
+          fastapi: v2.6.0.dev1
```

Comment:
fastapi: v2.6.0.dev1? is ok?

Reply:
yes it is, dev and dev1 are synonymous, because there is no dev0 usually.